packages/ui-scripts/package.json

```diff
@@ -19,8 +19,8 @@
   "scripts": {
     "lint": "ui-test --lint",
     "lint:fix": "ui-test --lint --fix",
-    "build": "tsc -p tsconfig.build.json",
```

Comment:
shouldn't we keep the build scripts? (there was another place where it was deleted)

Reply:
I haven't deleted the build scripts, rather just renamed them so they won't be rebuilded when we run `yarn run build`.